test/cmdlineTests.sh

```diff
@@ -533,6 +588,36 @@ printTask "Testing assemble, yul, strict-assembly and optimize..."
     test_solc_assembly_output "{ let x := 0 }" "{ { } }" "--strict-assembly --optimize"
 )
 
+printTask "Testing the eqivalence of --experimental-via-ir equivalence and a two-stage compilation..."
```

Comment:
This sentence phrasing has problems :D

Reply:
True, not sure how the second `equivalence` got in there :)

Reply:
Fixed.